includes/container/config.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+use Elementor\Core\Isolation\Wordpress_Adapter;
+use Elementor\Core\Isolation\Wordpress_Adapter_Interface;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+return [
+	Wordpress_Adapter_Interface::class => DI\autowire( Wordpress_Adapter::class ),
```

Comment:
```suggestion
	Wordpress_Adapter_Interface::class => DI\autowire( Wordpress_Adapter::class ),
```

`autowire` means it should be automatic, am I wrong?  here you just `bind` and interface to implementation

Reply:
https://php-di.org/doc/php-definitions.html#autowired-objects - DI\autowire tells the container to `autowire` when resolving an interface